toolchain/sem_ir/file.cpp

```diff
@@ -670,6 +671,9 @@ auto GetExprCategory(const File& file, InstId inst_id) -> ExprCategory {
       case TemporaryStorage::Kind:
       case ValueAsRef::Kind:
         return ExprCategory::EphemeralRef;
+
+      case ImportDecl::Kind:
+        CARBON_FATAL() << "Not valid in an expression.";
```

Comment:
Done, although after debugging https://github.com/carbon-language/carbon-lang/pull/4089 I have apprehension about this: it feels like it makes it really difficult to debug issues, that we don't do more to prevent calls here.